pkg/scheduler/core/generic_scheduler.go

```diff
@@ -208,8 +208,9 @@ func (g *genericScheduler) Schedule(ctx context.Context, state *framework.CycleS
 		}
 	}
 	trace.Step("Running postfilter plugins done")
-	metrics.DeprecatedSchedulingAlgorithmPredicateEvaluationSecondsDuration.Observe(metrics.SinceInSeconds(startPredicateEvalTime))
-	metrics.DeprecatedSchedulingDuration.WithLabelValues(metrics.PredicateEvaluation).Observe(metrics.SinceInSeconds(startPredicateEvalTime))
+	metrics.NodeFitEvaluationSeconds.Observe(metrics.SinceInSeconds(startNodeFitEvalTime))
```

Comment:
lets not introduce a new metric in this PR and focused on fixing the regression.